File: projects/tgs_fanj/tgs_反击/Script/lua/Dynamic/室3盒子.lua
--打开触发
function OnTurnOn(uSource, uDest)
  M_MapIceMonster(70, '室3四臂金刚', false);
  M_MapIceMonster(70, '室3护卫武士', false);
  M_MapboNotHItMonster(70, '室3四臂金刚', false);
  M_MapboNotHItMonster(70, '室3护卫武士', false);
  --打开室2门
  M_MapChangeDynamicobject(70, '室3墙壁', 2);
end;

--关闭触发
function OnTurnOff(uSource)
  M_MapIceMonster(70, '室3四臂金刚', true);
  M_MapIceMonster(70, '室3护卫武士', true);
  M_MapboNotHItMonster(70, '室3四臂金刚', true);
  M_MapboNotHItMonster(70, '室3护卫武士', true);
end;

File: server_code/debug/tgs1000/Script/Dynamic/室3盒子.pas
{******************************************************************************
室2塔死亡打开室2门
******************************************************************************}

   //打开

procedure OnTurnOn(uSource, uDest: integer);
begin
  MapIceMonster('王陵3层', '室3四臂金刚', false);
  MapIceMonster('王陵3层', '室3护卫武士', false);
  //打开室2门
  MapChangeDynamicobject('王陵3层', '室3墙壁', 'dos_Openning');

end;
 //复活

procedure OnTurnOff(uSource: integer);
begin
  MapIceMonster('王陵3层', '室3四臂金刚', true);
  MapIceMonster('王陵3层', '室3护卫武士', true);
end;

